packages/next/client/app-next.js

```diff
@@ -1,10 +1,12 @@
 import { appBootstrap } from './app-bootstrap'
 
-appBootstrap(() => {
-  // Include app-router and layout-router in the main chunk
-  import('next/dist/client/components/app-router.js')
-  import('next/dist/client/components/layout-router.js')
+// Include app-router and layout-router in the main chunk
+require('next/dist/client/components/app-router')
+require('next/dist/client/components/layout-router')
+require('react')
+require('next/dist/compiled/react-server-dom-webpack/client')
```

Comment:
These has to be executed after `appBootstrap`, need to move them into the callback.